live_view_studio/lib/live_view_studio_web/live/light_live.ex

```diff
@@ -0,0 +1,95 @@
+defmodule LiveViewStudioWeb.LightLive do
+  @moduledoc """
+    #A LiveView module needs to define three callback functions:
+    *mount assigns the initial state of the LiveView process
+    *handle_event changes the state of the process
+    *render renders a new view for the newly-updated state
+  """
+  use LiveViewStudioWeb, :live_view
+
+  @doc """
+    #the mount callback it's the first callback that's invoked when the request comes in throught the router
+    *params is a map containing the current query params, as well as any router parameters
+    *session is a map containing private session data
+    *socket is a struct representing the websocket connection
+  """
+  def mount(_params, _session, socket) do
```

Comment:
@keliumJU what does a mount do?

Reply:
mount, being the first function that is called in the request, is responsible for opening a connection through websockets, where we have the freedom to create an initial state.